### PR TITLE
Use prettier for code formatting

### PR DESCRIPTION
**Pull request by JuliaGerasymenko**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+doc/maintainers.md
```
Comment (belochub) [inline .prettierignore]: This should not be ignored.

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+node_modules
```
Comment (belochub) [inline .prettierignore]: This is redundant.

File: doc/maintainers.md [hunk 3]
```diff
@@ -51,7 +51,6 @@
 [`metarhia/sandboxed-fs`]: https://github.com/metarhia/sandboxed-fs
 [`metarhia/tickplate`]: https://github.com/metarhia/tickplate
 [`metarhia/tools`]: https://github.com/metarhia/tools
-
```
Comment (belochub) [inline doc/maintainers.md]: I think we can replace these empty lines with comments here:
```md
[//]: # (links to npm packages:)
```

File: doc/maintainers.md [hunk 4]
```diff
@@ -76,7 +75,6 @@
 [`sandboxed-fs`]: https://npmjs.com/sandboxed-fs
 [`tickplate`]: https://npmjs.com/tickplate
 [`@metarhia/tools`]: https://www.npmjs.com/@metarhia/tools
-
```
Comment (belochub) [inline doc/maintainers.md]: Similarly here:
```md
[//]: # (links to maintainers' GitHub profiles:)
```

File: .prettierignore (hunk 1 above)
Comment (JuliaGerasymenko) [reply .prettierignore]: Prettier changes the line 30 to `[`metarhia/metarhia`]: https://github.com/metarhia/Metarhia
`

File: doc/maintainers.md (hunk 3 above)
Comment (JuliaGerasymenko) [reply doc/maintainers.md]: Prettier changes this line to [//]: # 'links to npm packages:'

Comment (belochub) [reply doc/maintainers.md]: @JuliaGerasymenko, that is fine.

**Comment by belochub** [comment]
Landed in https://github.com/metarhia/Metarhia/commit/9511957e0ef531f8d0813dccf4987378b1c69718.

**Pull request closed without merging**